File: tests/scripting_js_faithful/10_set_members.js
// Faithful copy of Lua test 10_set_members.lua
// Expected: n
redis.call('DEL', KEYS[1]);
var n = Number(ARGV[1] || 100);
for (var i = 1; i <= n; i += 1) {
  redis.call('SADD', KEYS[1], 'm' + i);
}
var members = redis.call('SMEMBERS', KEYS[1]);
return members.length;

File: tests/scripting/10_set_members.lua
-- Populate a set and count members.
redis.call('DEL', KEYS[1]);
local n = tonumber(ARGV[1]) or 100;
for i = 1, n do
  redis.call('SADD', KEYS[1], 'm' .. i);
end
local members = redis.call('SMEMBERS', KEYS[1]);
return #members;
